ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGenSchemaUtil.java

```diff
@@ -23,12 +23,12 @@
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.apache.kafka.connect.data.Struct;
 
-final class SchemaUtil {
+public final class DataGenSchemaUtil {
```

Comment:
Renamed this class so as to not collide with the other `SchemaUtil` that is already public.